node/src/accountant/db_big_integer/big_int_db_processor.rs

```diff
@@ -253,28 +276,33 @@ impl<'a, T: TableNameDAO> BigIntSqlConfig<'a, T> {
     }
 }
 
-//to be able to display things that implement ToSql
-pub trait ExtendedParamsMarker: ToSql + Display {}
+// This is a native Rusqlite format for data supply to their interface
+type RusqliteParamPair<'params> = (&'params str, &'params dyn ToSql);
+// This is a representation of the above, plus providing a way to display the values (not possible natively)
+type DisplayableRusqliteParamPair<'params> = (&'params str, &'params dyn DisplayableParamValue);
 
-macro_rules! impl_of_extended_params_marker{
+// To be able to display things that implement ToSql
+pub trait DisplayableParamValue: ToSql + Display {}
+
+macro_rules! implement_displayable_param_value {
     ($($implementer: ty),+) => {
-        $(impl ExtendedParamsMarker for $implementer {})+
+        $(impl DisplayableParamValue for $implementer {})+
     }
 }
 
-impl_of_extended_params_marker!(i64, &str, Wallet);
+implement_displayable_param_value!(i64, &str, Wallet);
```

Comment:
I'd prefer if we directly implement it over the types. It'll remove half the lines.

```rust
impl DisplayableParamValue for i64 {}
impl DisplayableParamValue for &str {}
impl DisplayableParamValue for Wallet {}
```